tests/test_idpi/test_curl.py

```diff
@@ -37,12 +25,10 @@ def test_curl(data_dir):
     rlat = get_grid_coords(nj, lat_min, dlat, "y") * deg2rad
     total_diff = TotalDiff(dlon * deg2rad, dlat * deg2rad, ds["HHL"])
 
-    with print_time("curl"):
```

Comment:
did we not acre about the times here anymore?

Reply:
@cosunae requested this to be removed in another PR